### DISCUSSION TITLE
How often do icons get added?

### DISCUSSION THREAD

**Discussion by Brian-Pob** [devicons/devicon#1710] [General]
At the time of posting, the last commit was made in March of 2022. Is this project still being maintained?

**Comment by PantheraRed** [comment]
@Brian-Pob, please checkout [`develop`](https://github.com/devicons/devicon/tree/develop) branch. If you have, please clarify what you mean by still being maintained. Thank you.

**Reply by Brian-Pob** [reply]
My apologies. I just looked at the main branch and saw the last commit was added a year ago. I didn't realize there was a dev branch.

I also noticed that some recent technologies (ex. [Vite](https://vitejs.dev/), [SolidJS](https://www.solidjs.com/)) don't yet have logos on the site.

Is there a time frame for when new icons get added?

**Reply by PantheraRed** [reply]
@Brian-Pob, here [`wiki`](https://github.com/devicons/devicon/wiki/Release-strategy,-conventions,-preparation-and-execution) page discussing release strategy. Feel free to join community's Discord server for more info.

**Reply by Snailedlt** [reply]
We've wanted to put out a release for a few months now actually, but we've had several issues with the release, and little time to fix them. Hopefully the next release will be soon, but there is no ETA. Once the next release comes out however, we want to continue with sub-weekly or at least monthly releases
